static/app/views/issueList/overview.tsx

```diff
@@ -809,56 +803,10 @@ class IssueListOverview extends Component<Props, State> {
   listener = GroupStore.listen(() => this.onGroupChange(), undefined);
 
   onGroupChange() {
-    const {actionTakenGroupData} = this.state;
```

Comment:
Moved most of this logic from `onGroupChange()` to the `onActionTaken()` callback since we wait to call that after the groups have been updated